packages/flutter/lib/src/foundation/core_web.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'assertions.dart';
+import 'core_stub.dart' as core;
+import 'platform.dart';
+
+///
+TargetPlatform get defaultTargetPlatform {
+  TargetPlatform result = TargetPlatform.android;
+  if (debugDefaultTargetPlatformOverride != null)
+    result = debugDefaultTargetPlatformOverride;
+  if (result == null) {
+    throw FlutterError(
+      'Unknown platform.\n'
+      'Platform was not recognized as a target platform. '
+      'Consider updating the list of TargetPlatforms to include this platform.'
+    );
+  }
+  return result;
```

Comment:
you can simplify this whole method to `=> debugDefaultTargetPlatformOverride ?? TargetPlatform.android`.

But I don't know that that's the right answer. We should still detect iOS vs Android when we're in a mobile browser.